Description: Use pycodestyle instead of pep8
 In buster pep8 was a symlink to pycodestyle, in bullseye it is gone completely.
Author: Peter Michael Green <plugwash@debian.org>

--- python-cliapp-1.20180812.1.orig/check
+++ python-cliapp-1.20180812.1/check
@@ -5,5 +5,5 @@ set -eu
 python -m CoverageTestRunner --ignore-missing-from=without-tests
 python3 -m CoverageTestRunner --ignore-missing-from=without-tests
 rm -f .coverage
-pep8 cliapp
+pycodestyle cliapp
 # PYTHONPATH=. pylint --rcfile=pylint.conf cliapp *.py test-plugins/*.py
